test/test_operations.py

```diff
@@ -1006,6 +1006,19 @@ def func(a, b):
     b = torch.ones([2, 2])
     self.runAtenTest((a, b), func)
 
+  def test_multi_view(self):
+
+    def func(x):
+      a1, b1 = x.chunk(2)
+      a2, b2 = x[0:1], x[1:2]
+      a3, b3 = x[0].unsqueeze(0), x[1].unsqueeze(0)
+      a4, b4 = x[0, None], x[1, None]
+      return a1.squeeze(), b1.squeeze(), a2.squeeze(), b2.squeeze(), a3.squeeze(
```

Comment:
What will this return? I'm curious on how squeezing a [1] tensor works.... just another [1] tensor?

Reply:
it actually will return a scalar
```
>>> import torch
>>> x = torch.randn([2])
>>> x
tensor([-0.0293, -0.5956])
>>> x[0:1]
tensor([-0.0293])
>>> x[0:1].squeeze()
tensor(-0.0293)
```

Reply:
Perfect!